src/platforms/twitter.md

```diff
@@ -0,0 +1,66 @@
+# Twitter
+
+The Rust project has a number of official Twitter accounts, credentials for which are
+currently maintained by the infrastructure team.
+
+* [`@rustlang`](https://twitter.com/rustlang)
+* [`@RustStatus`](https://twitter.com/RustStatus)
+* [`@cratesiostatus`](https://twitter.com/cratesiostatus)
+* [`@rustasync`](https://twitter.com/rustasync)
+
+## Twitter Guidelines
+
+The project runs the Twitter account [@rustlang]
+The account is handled by a small team of volunteers.
+
+The account will mostly tweet links to the Rust blog and Rust Insiders blog.
+Additionally it will retweet:
+
+* links to blog posts about Rust, retweeting the original author if possible
+* questions about Rust, so all followers can help
+* Meetup or conference announcements
+* announcements of new Rust projects
+* anything else relevant
+
+We will not retweet:
+
+* content that bashes other programming languages/projects or is otherwise unconstructive in its discussion of language/tech choice
+* Personal announcements ("Today I start my job at $COMPANY writing Rust")
+* Learning Rust updates ("Today I started to learn Rust")
+
+The Direct Messages are open to everyone.
+If someone wants something retweeted, they should send the tweet via DM.
+The vast majority of these things should be retweeted, keeping it to the above rules.
+Requests of an author via DM or Tweet to not retweet something will be honored.
+
+Additionally account handlers may look through the [#rustlang](https://twitter.com/hashtag/rustlang?src=hashtag_click) hashtag for noteworthy content.
+
+The account will only follow a small number of Project-owned/related Twitter accounts.
+At the time of writing (February 2022) this is only [@cratesiostatus](https://twitter.com/cratesiostatus) and [@rust_foundation](https://twitter.com/rust_foundation).
```

Comment:
Just a note that this is still true and can be updated if there is an opportunity to do so.